Makefile

```diff
@@ -45,7 +45,7 @@ busted: dependencies $(ROVER) ## Test Lua.
 	@- luacov
 
 nginx:
-	@ ($(NGINX) -V 2>&1 | grep -e '--with-ipv6' > /dev/null) || (>&2 echo "$(NGINX) `$(NGINX) -v 2>&1` does not have ipv6 support" && exit 1)
+	@ ($(NGINX) -V 2>&1) > /dev/null
```

Comment:
I'm a bit confused by this @mikz 
We were checking that unnecessarily before, then?

Reply:
It was necessary in nginx <= 1.12. Because it did not offer ipv6 support by default. 1.13 does.

Reply:
:+1: